docs/source/features/packaging_output_models.md

```diff
@@ -138,8 +141,48 @@ If not specified, Olive will not package artifacts.
           The version for this data asset. This is `1` by default.
         * `description [str]`
           The description for this data asset. This is `None` by default.
+      * `AzureMLDeployment`
+        * `model_name [str]`:
+          The model name when registering your output model to your Azure ML workspace. `olive-deployment-model` by default
+        * `model_version [int | str]`:
+          The model version when registering your output model to your Azure ML workspace. Please note if there is already a model with the same name and the same version in your workspace, this will override your existing registered model. `1` by default.
+        * `description [str]`
+          The description for this model registration. This is `None` by default.
+        * `model_package [ModelPackageConfig]`:
+          The configurations for model packaging.
+          * `target_environment [str]`:
+            The environment name for the environment created by Olive. `olive-target-environment` by default.
+          * `target_environment_version [str]`
+            The environment version for the environment created by Olive. Please note if there is already an environment with the same name and the same version in your workspace, your existing environment version will plus 1. This `target_environment_version` will not be applied for your environment. `None` by default.
+          * `inferencing_server [InferenceServerConfig]`
+            * `type [str]`
+              The targeted inferencing server type. `AzureMLOnline` or `AzureMLBatch`.
+            * `code_folder [str]`
+              The folder path to your scoring script.
+            * `scoring_script [str]`
+              The scoring script name.
+          * `base_environent_id [str]`
```

Comment:
So for this one, we expect the user to have already created base environment beforehand?

Reply:
Yes, user still needs to provide a base environment.

Reply:
We may be able to support to automatically build one. This might be similar to task 27769. I will add this feature when I'm working on 27769.

Reply:
Sounds good to me. Thanks!